app/src/main/java/com/omen273/crossLingo/MainActivity.kt

```diff
@@ -340,6 +360,7 @@ class MainActivity : AppCompatActivity() {
                     }
                     try {
                         val adapter = tableLayout.adapter as TableAdapter
+                        Log.d("ERROR", "name: $name")
```

Comment:
Please remove it and Log import is it is possible

Reply:
removed, `Log` is still needed